spec/services/invoices/create_service_spec.rb

```diff
@@ -429,8 +436,8 @@
         aggregate_failures do
           expect(result).to be_success
 
-          expect(result.invoice.fees.first.properties['to_date']).to eq (timestamp - 1.day).to_date.to_s
-          expect(result.invoice.fees.first.properties['from_date']).to eq (timestamp - 1.year).to_date.to_s
+          expect(result.invoice.fees.first.properties['to_datetime']).to match_datetime((timestamp - 1.day).end_of_day)
+          expect(result.invoice.fees.first.properties['from_datetime']).to match_datetime((timestamp - 1.year).beginning_of_day)
```

Comment:
Layout/LineLength: Line is too long. [128/120] (https://rubystyle.guide#max-line-length)